spec/controllers/account_price_group_members_controller_spec.rb

```diff
@@ -3,30 +3,64 @@
 describe AccountPriceGroupMembersController do
   render_views
 
+  let(:price_group) { @authable.price_groups.create(FactoryGirl.attributes_for(:price_group)) }
+
   before(:all) { create_users }
 
   before(:each) do
     @authable = FactoryGirl.create(:facility)
-    @price_group = @authable.price_groups.create(FactoryGirl.attributes_for(:price_group))
   end
 
-  # allow admins
-  # allow directors
-  # allow staff
-  # deny guests
+  shared_examples "facility price group restrictions" do
+    context "with a facility specific price group" do
+      it_should_allow_all [:admin, :facility_admin, :director] do
+        successful_action_expectations
+      end
+
+      it_should_deny_all [:staff, :senior_staff]
+
+      context "with account specific roles" do
+        before { @authable = @account }
+
+        it_should_deny_all [:owner, :purchaser, :business_admin]
+      end
+    end
+  end
+
+  shared_examples "global price group restrictions" do
+    context "with a global price group" do
+      let(:price_group) { FactoryGirl.create(:price_group, :cancer_center) }
+
+      it_should_allow :admin do
+        successful_action_expectations
+      end
+
+      it_should_deny_all [:facility_admin, :director, :staff, :senior_staff]
+
+      context "with account specific roles" do
+        before { @authable = @account }
+
+        it_should_deny_all [:owner, :purchaser, :business_admin]
+      end
+    end
+  end
 
   context "new" do
     before :each do
       @method=:get
       @action=:new
-      @params={ :facility_id => @authable.url_name, :price_group_id => @price_group.id }
+      @account=create_nufs_account_with_owner
```

Comment:
Can we clean up some of this formatting and add spaces around the `=`? I'd rather change the whole file to match our organizational styles than continue to adhere to older stylings for consistency. (But I'd only worry about the files that you're currently touching).